src/Humanizer/Localisation/NumberToWords/BrazilianPortugueseNumberToWordsConverter.cs

```diff
@@ -26,7 +26,7 @@ public override string Convert(int number, GrammaticalGender gender)
             if ((number / 1000000000) > 0)
             {
                 // gender is not applied for billions
-                parts.Add(number / 1000000000 > 2
+                parts.Add(number / 1000000000 == 1
                     ? string.Format("{0} bilhões", Convert(number / 1000000000, GrammaticalGender.Masculine))
                     : string.Format("{0} bilhão", Convert(number / 1000000000, GrammaticalGender.Masculine)));
```

Comment:
You can use interpolated string:
```c#
var genderMark = number / 1000000000 == 1 ? "ões" : "ão";
parts.Add($"{Convert(number / 1000000000, GrammaticalGender.Masculine)} bilh{genderMark}");
```

And you can use something similar for millions (or even make it a private static method).